Runtime/WebViewPanel.cs

```diff
@@ -87,7 +88,7 @@ public void SetVisible(bool visible)
         // Builds RPM website URL for partner with given parameters.
         private string GetUrlFromConfig()
         {
-            var partnerSubdomain = ReadyPlayerMeSettings.GetPartnerSubdomain();
+            var partnerSubdomain = CoreSettings.PartnerSubDomainSettings.Subdomain;
```

Comment:
Just need to update SubDomain to subdomain
```suggestion
            var partnerSubdomain = CoreSettings.PartnerSubdomainSettings.Subdomain;
```